src/views/Terms.svelte

```diff
@@ -1,9 +1,164 @@
-<h1 class="py-4">Terms of use</h1>
+<script>
+import { tick } from 'svelte'
 
+export let params
+
+$: scrollToSection(params.section)
+
+async function scrollToSection(section) {
+  
```

Comment:
```suggestion
```

Reply:
unless the whitespace was deliberate

Reply:
The whitespace was deliberate (a blank line to make the comment stand out, and spaces to have the indentation match the correct indentation for that code block, to help diffs line up changes better by maintaining correct indentation even on blank lines).

However, we can certainly remove that if you like it better that way.

Reply:
nope, that's good then.